### PR TITLE
FPLA-121: Extend search cases functionality

### PR DESCRIPTION
**Pull request by rebeccabrennan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-121

### Change description ###
Search functionality extended to include Case name, CaseID (now working), State and Date submitted whilst Case creation date search removed.

### CODE REVIEW

File: ccd-definition/SearchResultFields.json [hunk 1]
```diff
@@ -9,22 +9,29 @@
...
+    "CaseFieldID": "[STATE]",
+    "Label": "State",
     "DisplayOrder": 3
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "[CREATED_DATE]",
-    "Label": "Case creation date",
+    "CaseFieldID": "caseLocalAuthority",
```
Comment (geek-soft) [inline ccd-definition/SearchResultFields.json]: Is that requirement to have different fields ordering? If not I suggest having one order of search fields so user does not have to learn location every time.

File: ccd-definition/AuthorisationCaseField.json [hunk 2]
```diff
@@ -258,13 +258,27 @@
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "dateSubmittedOnly",
```
Comment (swalker125) [inline ccd-definition/AuthorisationCaseField.json]: can you make it clear that this ID is for searching / filtering. Only means nothing to me

Comment (rebeccabrennan) [reply ccd-definition/AuthorisationCaseField.json]: searchAndFilterDate? 
dateForSearchAndFilter?
searchFilterDate?

Comment (rebeccabrennan) [reply ccd-definition/AuthorisationCaseField.json]: dateSubmitted -> dateAndTimeSubmitted
dateSubmittedOnly -> dateSubmitted

**Pull request merged**